src/interfaces/protocols/handlers/protocols-configure.ts

```diff
@@ -40,7 +39,7 @@ export class ProtocolsConfigureHandler implements MethodHandler {
       method    : DwnMethodName.Configure,
       protocol  : message.descriptor.protocol
     };
-    const existingMessages = await this.messageStore.query(tenant, query) as ProtocolsConfigureMessage[];
+    const existingMessages = await this.storageController.queryMessageStore(tenant, query) as ProtocolsConfigureMessage[];
```

Comment:
similar to [src/interfaces/messages/handlers/messages-get.ts:39](https://github.com/TBD54566975/dwn-sdk-js/pull/338/files#diff-5d5c842d6507a8df7c2f87057043b86e17d4fc15a5735e010e2c3c3d79c765a0R39), perhaps we should name this `queryMessages`?

Reply:
Addressed in https://github.com/TBD54566975/dwn-sdk-js/pull/338/commits/8a39de45aaa23185e406373a304c4ee9e4d68730